shared/docker/app/app-php.ini

```diff
@@ -0,0 +1,10 @@
+log_errors = On
```

Comment:
Is there a reason we can't use this for service-pdf? Does the opcache stuff mean it's unsuitable for a non-web app?

Reply:
OK, by using opcache in pdf I have now made them use the same file